src/redux/uniqueTokens.ts

```diff
@@ -244,34 +238,50 @@ export const fetchUniqueTokens = (showcaseAddress?: string) => async (
       type: UNIQUE_TOKENS_CLEAR_STATE_SHOWCASE,
     });
   }
-  const { network } = getState().settings;
+  const { network: currentNetwork } = getState().settings;
   const accountAddress = showcaseAddress || getState().settings.accountAddress;
   const { accountAssetsData } = getState().data;
   const { uniqueTokens: existingUniqueTokens } = getState().uniqueTokens;
   const shouldUpdateInBatches = isEmpty(existingUniqueTokens);
-
-  let shouldStopFetching = false;
-  let page = 0;
   let uniqueTokens: UniqueAsset[] = [];
 
-  const fetchPage = async () => {
+  const fetchNetwork = async (network: Network) => {
+    let shouldStopFetching: boolean = false!;
```

Comment:
what's the `!` here for?

Reply:
ah this is leftover ts from when was using a timeout, will rm